banzai_nres/pdf.py

```diff
@@ -26,14 +26,16 @@ def do_stage(self, image):
 
         phoenix_loader = phoenix.PhoenixModelLoader(self.runtime_context)
         template = phoenix_loader.load(image.classification)
-        v_over_c_plus_one = 1 + image.meta['RV']/constants.c.to(units.m/units.s).value
-
+        v_over_c_plus_one = 1 + (image.meta['RV'] - image.meta['BARYCORR'])/constants.c.to(units.m/units.s).value
+        wavelength_correction = 1/v_over_c_plus_one  # we will multiply the science frame wavelengths
```

Comment:
pep8 is technically to have space around operators. I'm a little surprised the code style check didn't catch this. There are a handful of these throughout the PR that should be fixed.